src/actions/transactionActions.ts

```diff
@@ -29,9 +35,26 @@ const setReceiverAddress = (receiverAddress: string) => ({
   type: TransactionActionTypes.SET_RECEIVER_ADDRESS
 });
 
+const createTransactionStatus = (initialTransaction: TransanctionStatus) => {
+  includeNewTransaction(initialTransaction);
```

Comment:
Seems that currently we only append this stuff and never read it, right? I'd suggest removing this for the current PR cause it adds no value at all and adding a separate issue to make it work properly.